aten/src/ATen/core/typeid.h

```diff
@@ -470,7 +468,7 @@ CAFFE_DECLARE_KNOWN_TYPE(7, double)
 CAFFE_DECLARE_KNOWN_TYPE(8, at::ComplexHalf)
 CAFFE_DECLARE_KNOWN_TYPE(9, std::complex<float>)
 CAFFE_DECLARE_KNOWN_TYPE(10, std::complex<double>)
-// 10 = undefined type id
+// 11 = undefined type id
```

Comment:
Hm, how come?

Reply:
It was a bug. 10 is std::complex<double>